htdocs/kernel/menusitems.php

```
@@ -0,0 +1,263 @@
+        if (!isset(self::$languageFilesIncluded[$fileinc])) {
+            if (file_exists($fileinc)) {
+                include_once $fileinc;
+                self::$languageFilesIncluded[$fileinc] = true;
+            } else {
+                // Fallback to English if language file not found
+                $fallback = XOOPS_ROOT_PATH . '/modules/system/language/english/menus/menus.php';
+                if ($fileinc !== $fallback && !isset(self::$languageFilesIncluded[$fallback]) && file_exists($fallback)) {
+                    include_once $fallback;
+                    self::$languageFilesIncluded[$fallback] = true;
+                }
+            }
```

PHPMD (2.15.0)

[error] The method __construct uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)